src/pyrad_proc/pyrad/io/io_aux.py

```diff
@@ -343,7 +344,7 @@ def get_fieldname_pyart(datatype):
     return field_name
 
 
-def get_file_list(datadescriptor, starttime, endtime, cfg, scan=None):
+def get_file_list(datadescriptor, starttime, endtime, cfg, scan=None, lteconvention = True):
```

Comment:
let's include a variable in the config file called "path_convention" with possible values (for the moment): LTE, MCH.
You will then inspect cfg['path_convention'] instead of lteconvention.